src/compo/tropess_co_nc2ioda.py

```diff
@@ -19,14 +19,9 @@
 import math
 from numpy import log as ln
 
-IODA_CONV_PATH = Path(__file__).parent/"@SCRIPT_LIB_PATH@"
-if not IODA_CONV_PATH.is_dir():
-    IODA_CONV_PATH = Path(__file__).parent/'..'/'lib-python'
-sys.path.append(str(IODA_CONV_PATH.resolve()))
-
-import ioda_conv_engines as iconv
+import lib_python.ioda_conv_engines as iconv
```

Comment:
we should really change this to be `import pyiodaconv.ioda_conv_engines` or something similar. Like I've said before `lib_python` is not a sufficient name, it would be like '#include cpp`

Reply:
@CoryMartin-NOAA it's doable. I will need to change the directory name from lib_python to something else. Let me give it a look.

Reply:
@CoryMartin-NOAA fully agree lump it in this one or do in a subsequent PR could be considered